src/js/services/opencap.service.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+(function() {
+  angular.module('bitcoincom.services').factory('opencapService', function($q, $http) {
+    function getAddress(alias) {
+      let aliasData = validateAlias(alias);
+      if (aliasData.username === '' || aliasData.domain === '') {
+        return $q((resolve, reject) => {
```

Comment:
Now :  `=>`  : lambda expression is not supported for old javascript engine.
Should be : 
```
function onQ() {
}
```